app/helpers/question_helper.rb

```diff
@@ -1,2 +1,9 @@
 module QuestionHelper
+  def question_header(object)
+    if object.new_record?
+      "Create New #{@question.test.title} Question"
```

Comment:
правильней использовать прешедший аргумент, а не глобальную переменную - меньше шансов ошибиться
```suggestion
      "Create New #{question.test.title} Question"
```